pyproject.toml

```diff
@@ -0,0 +1,17 @@
+[build-system]
+build-backend = "flit_core.buildapi"
+requires = ["flit_core >=3.8.0,<4"]
+
+[project]
+name = "UD-UM"
```

Comment:
not a great name for this. I would probably go with Arya's original `ud-compatibility`.

Reply:
it has to do with the folder name - I think I would have to rename the `ud-um` folder to `ud-compatibility` to do this - https://stackoverflow.com/questions/69230525/why-does-poetry-build-raise-moduleorpackagenotfound-exception

Reply:
This is of course up to @aryamccarthy but I think then that directory should also be renamed, since that's also a little terse.